test/integration/appDuplicate.spec.ts

```diff
@@ -10,10 +10,18 @@ test('duplicate app from home flow', async ({ page }) => {
   await homeModel.goto();
   const app = await homeModel.createApplication({ dom });
   await homeModel.goto();
-  await homeModel.duplicateApplication(app.name);
+  const duplicateApp = await homeModel.duplicateApplication(app.name);
   await homeModel.goto();
   await expect(page.locator(locators.toolpadHomeAppRow(`${app.name} (copy)`))).toBeVisible();
   await homeModel.duplicateApplication(app.name);
   await homeModel.goto();
   await expect(page.locator(locators.toolpadHomeAppRow(`${app.name} (copy 2)`))).toBeVisible();
+
+  await page.goto(`/_toolpad/app/${duplicateApp.id}/preview/pages`);
```

Comment:
the `ToolpadRuntime` model might be useful for this part
checkout out the `components` integration test, maybe this test could do something similar (if it's not working)

Reply:
Thanks! I'll check that out